### PR TITLE
Increase Size of Notified Pokemon

### PR DESCRIPTION
**Pull request by Falaris01**
Added the ability to scale the size of notified pokemon based on IV / Level / Pokemon notification settings.

image

image

### CODE REVIEW

**Comment by Falaris01** [comment]
Travis failed at git clone for whatever reason...

Is there a way to make it re-try now?

**Comment by Glennmen** [comment]
@Falaris01 I manually restarted Travis, there are some errors that need to be fixed.

**Comment by Falaris01** [comment]
Ok got them fixed :)

**Comment by 123FLO321** [comment]
This does not cover `Notify of Rarity`
Is that intentional?

**Comment by 123FLO321** [comment]
And If you click on them they're not in front any longer.
That's not necessarily a problem of this PR but would be nice and probably easy to fix.

**Comment by Falaris01** [comment]
Thanks, I actually missed rarity notification as I used to have that disabled by default :)

**Comment by Falaris01** [comment]
> And If you click on them they're not in front any longer.That's not necessarily a problem of this PR but would be nice and probably easy to fix.

Not sure if it's desired. While it is indeed very easy, a short test showed it's shortcomings.

With the increased icon size of notified pokemon the smaller icons might get stuck behind those and end up making the user need to zoom in to see what's behind.

image

**Comment by 123FLO321** [comment]
True maybe. 
But it worse the other way around since larger ones are probably more important

**Comment by Falaris01** [comment]
True, it might only be an issue with the safe icons anyway and could be submitted in another PR to try the changes out.

File: static/js/map.common.js [hunk 1]
```diff
@@ -1143,6 +1152,35 @@ function setupPokemonMarker(item, map, isBounceDisabled) {
...
+        notifiedPokemon.indexOf(item['pokemon_id']) > -1 ||
+        notifiedRarity.indexOf(item['pokemon_rarity']) > -1 ||
+        (notifyLevel > 0 && level >= notifyLevel)) {
+        return true
+    }
+
+    return false
+}
+
+function getIv(atk, def, sta) {
```
Comment (Glennmen) [inline static/js/map.common.js]: This function already exists in map.js, you should be able to call it.

**Review by Glennmen — CHANGES_REQUESTED** [review]

**Comment by Falaris01** [comment]
Thanks for pointing that out. 

I reworked it a little, it now adds to iconSize instead of using a new var.

**Comment by Falaris01** [comment]
Sorry for the mess, I'll work this out and add another PR once I'm done

**Pull request closed without merging**